airflow/providers/amazon/provider.yaml

```diff
@@ -499,6 +499,9 @@ hooks:
   - integration-name: Amazon Simple Storage Service (S3)
     python-modules:
       - airflow.providers.amazon.aws.hooks.s3
+  - integration-name: Minio Simple Storage Service (S3)
+    python-modules:
+      - airflow.providers.amazon.aws.hooks.minio
```

Comment:
I'm late to this discussion, why is the AWS provider now responsible for/owns a Minio hook? I know that they support the S3 API, but I'm not sure that warrants us owning and vending that code in the AWS provider package. It sets a bit of a precedent I think.

Reply:
> I'm late to this discussion, why is the AWS provider now responsible for/owns a Minio hook? I know that they support the S3 API, but I'm not sure that warrants us owning and vending that code in the AWS provider package. It sets a bit of a precedent I think.

Quite agree. It feels out of place here.

Reply:
110% agree with Niko here. We should not set this precedence that Amazon provider package can or will own MinIO hook or any other hooks outside of services under AWS. I understand that MinIO is specifically for using S3, but it is not part of AWS portfolio.

Reply:
I guess it calls for following the protocol of adding new provider
https://github.com/apache/airflow/blob/main/PROVIDERS.rst#accepting-new-community-providers

Reply:
Pretty much so.